src/containers/ModelPanel/index.tsx

```diff
@@ -112,11 +125,35 @@ const ModelPanel: React.FC<ModelPanelProps> = ({
         ),
     };
 
+    const handlePreviewDefaultColors = (colorSetting: ColorSetting) => {
+        if (!defaultUiSettingsApplied) {
+            setCurrentColorSetting({ currentColorSetting: colorSetting });
+        }
+    };
+
     return (
         <div className={styles.container}>
             <SideBarContents
                 mainTitle="Agents"
-                content={[contentMap[viewerStatus]]}
+                content={[
+                    <div key="molecules">
```

Comment:
is `molecules` maybe too specific a name for this?

Reply:
I wrote the first drafts of this a long time ago, guess I was feeling granular?